[TASK] Move content of "Data Types" (#1459)
TYPO3-Documentation/TYPO3CMS-Reference-Typoscript: Documentation/Functions/Wrap.rst

..  include:: /Includes.rst.txt
..  _data-type-wrap:

====
Wrap
====

..  confval:: wrap
    :name: data-type-wrap

    :Syntax: <...> \| </...>

    Used to wrap something. The vertical bar ("|") is the place, where
    your content will be inserted; the parts on the left and right of the
    vertical line are placed on the left and right side of the content.

    Spaces between the wrap-parts and the divider ("|") are trimmed off
    from each part of the wrap.

    If you want to use more sophisticated data functions, then you
    should use `stdWrap.dataWrap` instead of `wrap`.

>     A `wrap` is processed and rendered as the last of the other components of

Author: I think we should remove this sentence as really it is the implementation of the content element in question, which defines the execution order?

>     A `wrap` is processed and rendered as the last of the other components of

Reviewer: For me something like: "wrap was executed as one of the last properties" is enough.

>     A `wrap` is processed and rendered as the last of the other components of
>     a cObject.
> 

Author: ```suggestion
    A `wrap` is applied as one of the last of properties
    a cObject.

```

    ..  rubric:: Examples

    This will cause the value to be wrapped in a p-tag coloring the
    value red:

    ..  code-block:: typoscript
        :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript

>         page.10.stdWrap.wrap = <p class="bg-red"> | </p>

Reviewer: This example works, but does not relate to the "last wrap in content elements". This example explains how to use "wrap" in stdWrap" and here "wrap" is somewhere in the middle of all properties.

>         page.10.stdWrap.wrap = <p class="bg-red"> | </p>

Author: ```suggestion
        page.10.wrap = <p class="bg-red"> | </p>
```
